src/main/java/appeng/items/materials/ItemMultiMaterial.java

```diff
@@ -273,6 +275,19 @@ public boolean onItemUseFirst(final ItemStack is, final EntityPlayer player, fin
 
         if (player.isSneaking()) {
             final TileEntity te = world.getTileEntity(x, y, z);
+            if (te instanceof TileDrive drive) {
+                if (is != null && this.getType(is) == Upgrades.STICKY) {
+                    if (Platform.isServer()) {
+                        return drive.applyStickyToCells(player);
+                    }
+                }
+            } else if (te instanceof TileChest chest) {
+                if (is != null && this.getType(is) == Upgrades.STICKY) {
+                    if (Platform.isServer()) {
+                        return chest.applyStickyToCells(player);
+                    }
+                }
+            }
```

Comment:
nit: Move `lockCells`and `applyStickyToCells` into `IChestOrDrive` to combine both ifs into one.

Reply:
Resolved, see d64a646669a6487c86b053665044dae45a2d05be